pycbc/workflow/pegasus_files/local-site-template.xml

```diff
@@ -1,10 +1,10 @@
   <site handle="local" arch="x86_64" os="LINUX">
-    <directory  path="$PWD" type="shared-scratch" free-size="null" total-size="null">
-        <file-server  operation="all" url="file://$PWD">
+    <directory  path="$LOCAL_SITE_PATH/local-site-scratch" type="shared-scratch" free-size="null" total-size="null">
+        <file-server  operation="all" url="$LOCAL_SITE_URL/local-site-scratch">
```

Comment:
Yes, Karan though that we might run into problems if the scratch and output directories are the same. I raided the same concern as you did and he suggested just making the scratch directory a sub-directory of the storage directory. It's only one more level, and users probably should be poking around in the scratch directory, anyway. Dashboard will have correct links to the run directories.